src/app/views/PublishesNotices.svelte

```diff
@@ -0,0 +1,82 @@
+<script lang="ts">
+  import {formatTimestamp, thunks, type Thunk, type ThunkStatus} from "@welshman/app"
+  import {ctx, identity, uniq} from "@welshman/lib"
+  import {PublishStatus, type Connection} from "@welshman/net"
+  import cx from "classnames"
+  import {get} from "svelte/store"
+  import {fly} from "svelte/transition"
+  import SearchSelect from "src/partials/SearchSelect.svelte"
+  import AltColor from "src/partials/AltColor.svelte"
+  import {fuzzy} from "src/util/misc"
+  import {router} from "src/app/util"
+
+  export let selected: string[] = []
+
+  const searchConnections = fuzzy(Array.from(ctx.net.pool.data.keys()))
+
+  function getNotices(connections: Connection[]) {
+    return Object.values($thunks).filter(t =>
+      connections.some(cxn => get(t.status)[cxn.url]?.status),
+    )
+  }
```

Comment:
I'd also like to listen for NOTICE messages, which won't come through on thunks. Probably add this via the pool `init` event so we can get handlers on connections when they're set up.